src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -119,12 +120,18 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings)
             if (packageSources.Count < 1)
```

Comment:
Done

---
In reply to: [62937909](https://github.com/NuGet/NuGet.Client/pull/586#discussion_r62937909) [](ancestors = 62937909)